src/main/java/com/brandwatch/internship/fetchandstoretweets/entities/Mention.java

```diff
@@ -36,6 +38,21 @@ public Mention(long id, String idStr, String text, Date createdAt, String fromUs
         this.queryId = queryId;
     }
 
+    private static Mention tweetToMention(Tweet tweet, long queryId){
+        return new Mention(tweet.getId(), tweet.getIdStr(),tweet.getText(), tweet.getCreatedAt(), tweet.getFromUser(),
```

Comment:
Oh I see `idStr` is a `Tweet` field? Is it literally the id as a `String` or something different?

Reply:
I just saw this comment. I answered this question in the previous comment :-)